app/views/courses/ActivateLicensesModal.coffee

```diff
@@ -56,6 +55,10 @@ module.exports = class ActivateLicensesModal extends ModalView
       @state.set {
         unusedEnrollments: @prepaids.totalMaxRedeemers() - @prepaids.totalRedeemers()
       }
+      
+  onLoaded: ->
+    @prepaids.reset(@prepaids.filter((prepaid) -> prepaid.status() is 'available'))
```

Comment:
BTW reset does not trigger add/remove events, so line 54 won't be triggered by this.